### PR TITLE
Update example for semantic versioning

### PR DESCRIPTION
**Pull request by davidpiegza**
According to semver.org the initial release should start at `0.1.0` and then increment the minor version for subsequent releases (https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase). I updated the example in "Semantic versioning" section.

### CODE REVIEW

**Review by deivid-rodriguez — APPROVED** [review]
:+1: Thank you!

**Comment by davidpiegza** [comment]
@deivid-rodriguez Is an additional approval required to be able to merge? (I see  a "1 workflow awaiting approval" message). Maybe you can ping another maintainer if it's needed?

**Comment by deivid-rodriguez** [comment]
No, I'll merge after CI is done :+1:. Thank you!

**Pull request merged**
